tools/clang/test/CodeGenSPIRV/rich.debug.structured-buffer.hlsl

```diff
@@ -0,0 +1,50 @@
+// Run: %dxc -T ps_6_0 -E main -fspv-debug=rich
+
+struct S {
+    float  a;
+    float3 b;
+    int2 c;
+};
+
+struct T {
+    float  a;
+    float3 b;
+    S      c;
+    int2   d;
+};
+
+StructuredBuffer<S> mySBuffer1 : register(t1);
+StructuredBuffer<T> mySBuffer2 : register(t2);
+
+RWStructuredBuffer<S> mySBuffer3 : register(u3);
+RWStructuredBuffer<T> mySBuffer4 : register(u4);
+
+// CHECK: [[set:%\d+]] = OpExtInstImport "OpenCL.DebugInfo.100"
+// CHECK: [[S:%\d+]] = OpString "S"
+// CHECK: [[SB_S:%\d+]] = OpString "@type.StructuredBuffer.S"
+// CHECK: [[T:%\d+]] = OpString "T"
+// CHECK: [[SB_T:%\d+]] = OpString "@type.StructuredBuffer.T"
+// CHECK: [[RW_S:%\d+]] = OpString "@type.RWStructuredBuffer.S"
+// CHECK: [[RW_T:%\d+]] = OpString "@type.RWStructuredBuffer.T"
+// CHECK: [[temp0:%\d+]] = OpString "StructuredBuffer.TemplateParam"
+// CHECK: [[temp1:%\d+]] = OpString "RWStructuredBuffer.TemplateParam"
+
+// CHECK: [[S_ty:%\d+]] = OpExtInst %void [[set]] DebugTypeComposite [[S]] Structure
+// CHECK: [[SB_S_ty:%\d+]] = OpExtInst %void [[set]] DebugTypeComposite [[SB_S]] Class
```

Comment:
According to the spec:
> To represent a source language opaque type this instruction must have no *Members* operands, *Size* operand must be **DebugInfoNone** and *Name* must start with *@* symbol to avoid clashes with user defined names.

This CHECK shows that the name starts with @.   But it doesn't show that the size operand is DebugInfoNone. It also doesn't check that there are no **Members** operand.